src/components/mortgage-investment-compare/input-card.js

```diff
@@ -64,12 +104,12 @@ const InputCard = props => {
                 <b>Mortgage Option 1</b>
               </small>
               <InputGroup className='my-2'>
-                <Input
-                  name='options.option1.term'
+                <ValidatedNumberInput
+                  name='option1.term'
+                  onChange={onInputChange}
                   placeholder='Loan Term'
-                  type='number'
-                  value={props.option1.term}
-                  onChange={props.onInputChange}
+                  mustBeGreaterThan={0}
+                  value={props.input.option1.term}
```

Comment:
pass in `min={0}` so that we can prevent the user from going below 0 when using the arrows.